src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.tizen.cs

```diff
@@ -20,19 +20,20 @@ public static class DrawingViewService
 	public static ValueTask<Stream> GetImageStream(IList<PointF> points, Size imageSize, float lineWidth, Color strokeColor, Paint? background)
 	{
 		var image = GetBitmapForPoints(points, lineWidth, strokeColor, background);
+
 		if (image is null)
 		{
 			return ValueTask.FromResult(Stream.Null);
 		}
-		return new ValueTask<Stream>(Task.Run<Stream>(() =>
-		{
-			var resized = image.Resize(new SKImageInfo((int)imageSize.Width, (int)imageSize.Height, SKColorType.Bgra8888, SKAlphaType.Opaque), SKFilterQuality.High);
-			var data = resized.Encode(SKEncodedImageFormat.Png, 100);
-			var stream = new MemoryStream();
-			data.SaveTo(stream);
-			stream.Seek(0, SeekOrigin.Begin);
-			return stream;
-		}));
+
+		var resized = image.Resize(new SKImageInfo((int)imageSize.Width, (int)imageSize.Height, SKColorType.Bgra8888, SKAlphaType.Opaque), SKFilterQuality.High);
+		var data = resized.Encode(SKEncodedImageFormat.Png, 100);
```

Comment:
In my opinion, `Resize` is relatively heavy jobs and this thread is UI Thread so We should not block the thread. 
and `Resize` is thread-safe so I run it on worker thread.

Reply:
Ah - understood. Fixed ✅